Week03/lesson05/classwork/week3-relaxr-part2/css/style.css

```diff
@@ -0,0 +1,241 @@
+/*
+Project Name: Relaxr Blog Homework Lesson 5
+Client: Relaxr
+Author: Christophe Brumby
+Developer @GA in London
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+.alignright {
+  float: left;
+  padding: 0 0 10px 0; /* note the padding around a right floated image */
+  width: 50%;
+  text-align: right;
+}
+
+.alignleft {
+  float: right;
+  padding: 0 0 10px 20px; /* note the padding around a left floated image */
+  width: 50%;
+  text-align: left;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+  font-family: Open Sans, Helvetica, Arial, sans-serif;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+  width: 1100px;
+  margin: auto;
+  background-color: #f0efef;
```

Comment:
OK, I have followed your advice for the Startup Matchmaker top nav for instance.